geoips/plugins/modules/readers/viirs_sdr_hdf5.py

```diff
@@ -5,41 +5,44 @@
 
 This VIIRS reader is designed for reading the NPP/JPSS SDR HDF5 files.
 The input files are produced by CSPP Polar (CSPP RDR pipeline),
-and the read by satpy.
+and the read by hdf5.
 
-V1.1.0:  NRL-Monterey, Aug. 2024
+V2.0.0:  NRL-Monterey, 082025
 
 """
 
 # Python Standard Libraries
 import logging
 import os
+import re
+from itertools import chain
+from datetime import datetime, timedelta
 
 # Third-Party Libraries
 import h5py
 import numpy as np
-from pandas import DataFrame, date_range, to_datetime
-from scipy.interpolate import NearestNDInterpolator
 import xarray as xr
+from scipy.interpolate import NearestNDInterpolator
 
-# GeoIPS Libraries
-from geoips.plugins.modules.readers.utils.geostationary_geolocation import get_indexes
-from geoips.utils.context_managers import import_optional_dependencies
-
-# If this reader is not installed on the system, don't fail altogether, just skip this
-# import. This reader will not work if the import fails, and the package will have to be
-# installed to process data of this type.
+# geoips libraries
+from geoips.data_manipulations.corrections import apply_gamma
 
 LOG = logging.getLogger(__name__)
 
+try:
+    from lunarref.lib.liblunarref import lunarref
+
+    lunarref_lib = True
+except ImportError:
+    lunarref_lib = False
+    LOG.warning("Failed lunarref import, if needed, install it.")
+
+
 interface = "readers"
 family = "standard"
 name = "viirs_sdr_hdf5"
 source_names = ["viirs"]
 
-with import_optional_dependencies(loglevel="info"):
-    """Attempt to import a package and print to LOG.info if the import fails."""
-    import satpy
```

Comment:
Updates to viirs_sdr_hdf5.py removed satpy dependence (native geoips reader), and added bowtie correction.